src/main/java/seedu/duke/constants/Messages.java

```diff
@@ -59,4 +59,13 @@
     public static final String DECOR_BEFORE = "|  ";
     public static final String DECOR_AFTER = "  |";
     //SCHEDULE END
+
+    //HELP BEGIN
+    public static final String HELP_MESSAGE = "List of valid commands:\n1. module\n2. calendar\n3. food\n"
+            + "4. exit\nType help <COMMAND_NAME> to get a detailed description";
```

Comment:
this should be a method call instead, since we have classes that extend `Command` and constants that we check for in `parser`

Reply:
to be exact, this could be a method in `Ui` rather than a string, having added the commands and the descriptions, where a short description could be tagged with each command in the definition